mmv1/third_party/terraform/utils/utils.go.erb

```diff
@@ -477,3 +477,12 @@ func generateUserAgentString(d TerraformResourceData, currentUserAgent string) (
 
 	return currentUserAgent, nil
 }
+
+
+func PascalCase(s string) (string) {
```

Comment:
nitpick: since this only needs to work for the underscore delim, maybe `SnakeToPascalCase` or something will be more transparent

Reply:
ooh, yeah, that's better. Thanks for the suggestion!